Sources/Build/BuildPlan/BuildPlan+Test.swift

```diff
@@ -133,21 +136,34 @@ extension BuildPlan {
                 )
             }
 
+            let discoveryTargets: (target: SwiftTarget, resolved: ResolvedTarget, buildDescription: SwiftTargetBuildDescription)?
```

Comment:
ugh, tuples 😖 
ok to keep this way for now, since `generateDiscoveryTargets` wasn't changed here.

Reply:
👞🎺